src/main/java/org/openstreetmap/atlas/tags/SyntheticSyntheticRelationMemberTag.java

```diff
@@ -0,0 +1,26 @@
+package org.openstreetmap.atlas.tags;
+
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+import org.openstreetmap.atlas.tags.annotations.validation.Validators;
+
+/**
+ * Tag to indicate an entity is a synthetic relation member added during country slicing.
+ * <p>
+ * This is not an OSM tag.
+ *
+ * @author samg
+ */
+@Tag(synthetic = true)
+public enum SyntheticSyntheticRelationMemberTag
```

Comment:
Nit: Synthetic repeated twice.

Reply:
I waffled on this, the format seems to be `Synthetic[description]Tag` and the description in this case is `SyntheticRelationMember`-- i.e., it's a Synthetic Tag tracking Synthetic Relation Members. I went with this, but if there's strong feelings towards the other name I'm happy to do that.